modules/nodes-scheduler-taints-tolerations-removing.adoc

```diff
@@ -31,6 +31,55 @@ $ oc adm taint nodes ip-10-0-132-248.ec2.internal key1-
 ----
 node/ip-10-0-132-248.ec2.internal untainted
 ----
++
+[TIP]
```

Comment:
Let's remove this TIP because with GitOps, the customer would just remove the edit that added the taint and redeploy.